woof-code/rootfs-skeleton/usr/sbin/quicksetup

```diff
@@ -1185,10 +1185,10 @@ if [ "$SET_KEYBOARD" ];then
  if [ -f /etc/X11/xorg.conf ]; then
   sed -i "s%.*Option.*XkbLayout.*%	Option      \"XkbLayout\" \"${XKB_LAYOUT}\"%" /etc/X11/xorg.conf
  fi
- if [ -f ~/.Xwaylandrc ]; then
-  sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" ~/.Xwaylandrc
-  if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
-    sed -i "s%^XKB_DEFAULT_LAYOUT=.*%XKB_DEFAULT_LAYOUT=$XKB_LAYOUT%" $XDG_CONFIG_HOME/labwc/environment
+ if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
```

Comment:
What I meant is,

```
if [ -f ~/.Xwaylandrc ]; then
  ...
fi

if [ -f "$XDG_CONFIG_HOME/labwc/environment" ]; then
  ...
fi
```

~/.Xwaylandrc can exist if labwc is not present, and labwc can be present without Cage and .Xwaylandrc.